guacamole/src/main/webapp/layouts/fr-fr-azerty.json

```diff
@@ -31,6 +31,43 @@
 
         "Space" : " ",
 
+        "Back" : [{
```

Comment:
Keys must have only one definition - the JSON is invalid otherwise. You'll need to delete the old definitions having the same names (compare with [de-de-qwertz.json](https://github.com/glyptodon/guacamole-client/blob/master/guacamole/src/main/webapp/layouts/de-de-qwertz.json)).